bakerydemo/base/models.py

```diff
@@ -132,7 +100,7 @@ class Meta:
 class StandardPage(Page):
     """
     A generic content page. On this demo site we use it for an about page but
-    could be used for any type of page content.
+    it could be used for any type of page content.
```

Comment:
Perhaps we could be a little more explicit (now there isn't a mixin) saying

> ... it could be used for any type of page content that only needs a title, image, introduction and body field